TextChecker.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Graph_Visual_Laba_3
+{
+    public static class TextChecker
+    {
+        public static bool CheckString(string str)
+        {
+            bool flag = true;
+
+            if (String.IsNullOrWhiteSpace(str))
+            {
+                flag = false;
+                return flag;
+            }
+            int len = str.Length;
+            for (int i = 0; i < len; ++i)
+            {
+                char c = str[i];
+                if ((c < '0' || c > '9') && c != ',')
+                    flag = false;
+            }
+            return flag;
+        }
+
+        public static List<Tuple<string, string>> CheckFile(string path)
+        {
+            bool check_str = false;
```

Comment:
The same here. You should avoid needless variable, CleanCode says that it is a bad practice